### DISCUSSION TITLE
can not login

### DISCUSSION THREAD

**Discussion by fungmen** [matamorphosis/Scrummage#29] [Q&A] [answered]
hi 
can anyone help me how to login 
I just finish installing but I can not login

**Comment by matamorphosis** [comment, marked as answer]
Hi @fungmen,

Please refer to the wiki pages for installation guidance.
[Manual Installation](https://github.com/matamorphosis/Scrummage/wiki/Installation)
[Docker Installation](https://github.com/matamorphosis/Scrummage/wiki/Installation-using-Docker)

In summary, you create a user manually in the manual installation using the `Create_User.py` utility.  
In the docker installation, the admin password is generated upon running the container for the first time, refer to the below example output:
```console
user@linux:~$ docker run -p 5000:5000 ghcr.io/matamorphosis/scrummage:latest
---EXAMPLE OUTPUT---
 * Restarting PostgreSQL 12 database server
   ...done.
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0

----------------------------------------------------------------------------------------------------
This is the admin password, this will only be displayed on the first run of Scrummage:
<PASSWORD WILL BE DISPLAYED HERE>
----------------------------------------------------------------------------------------------------
...
```
